lib/httpoison/in_memory.ex

```diff
@@ -4,6 +4,9 @@ defmodule ElixirAuthGithub.HTTPoison.InMemory do
     code in ElixirAuthGithub. These are just functions that pattern match on the entries and return things in the way we expect, so that we can check the pipeline in ElixirAuthGithub.github_auth
   """
 
+  #hello world
+  #hello again
+
```

Comment:
All good apart from these comments. Are they there for a reason?

Reply:
I have no idea how this happened.

Reply:
fix'd